### PR TITLE
Use styles from site-shared/dash_design

### PR DESCRIPTION
**Pull request by isoos**
- Updated the variable tests to also use the dash_design files.
- I've commented out the two colors that this replaces for now. I'm going to remove them with a follow-up PR when we have more colors in the shared style sheet that we import.
- No effective color change for us.

### CODE REVIEW

File: pkg/web_css/lib/src/_variables.scss [hunk 1]
```diff
@@ -16,17 +16,19 @@
...
 
 :root {
+  @include dash_variables.light-theme;
+
   --pub-color-white:         #ffffff;
   --pub-color-snowWhite:     #fafafa; // slight deviation from official snow-white (fffafa)
   --pub-color-smokeWhite:    #f5f5f7; // slight deviation from official white-smoke (f5f5f5)
   --pub-color-bubblesBlue:   #e7f8ff; // slight deviation from official bubbles-blue (e7feff)
-  --pub-color-gunpowderGray: #4a4a4a; // hsl(0, 0%, 29%);
+  // --pub-color-gunpowderGray: #4a4a4a; // hsl(0, 0%, 29%);
```
Comment (sigurdm) [inline pkg/web_css/lib/src/_variables.scss:25]: Should this just be deleted?

**Review by sigurdm — APPROVED** [review]

File: pkg/web_css/lib/src/_variables.scss [hunk 2]
```diff
@@ -114,15 +116,17 @@
...
 
 .dark-theme {
+  @include dash_variables.dark-theme;
+
   --pub-color-darkGunmetal:    #1f262a; // close to #1d2026
   --pub-color-shadowBlack:     #373737;
   --pub-color-anchorBlack:     #41424c;
   --pub-color-nipponUltraBlue: #23607f;
-  --pub-color-gainsboro:       #dcdcdc;
+  // --pub-color-gainsboro:       #dcdcdc;
```
Comment (sigurdm) [inline pkg/web_css/lib/src/_variables.scss:125]: And this..

File: pkg/web_css/lib/src/_variables.scss (hunk 1 above)
Comment (isoos) [reply pkg/web_css/lib/src/_variables.scss:25]: I'd rather keep this for a short while, until we have a better consolidated view on how the naming + shared naming will work. But at the same time I can also just delete it.

Comment (sigurdm) [reply pkg/web_css/lib/src/_variables.scss:25]: As you prefer - just note that the usual policy is to not leave commented out code.

You can always find it in git history.

**Pull request merged**